code/modules/maps/planet_types/lore/uueoaesa.dm

```diff
@@ -0,0 +1,294 @@
+
+	desc += " The landing sites are located in the [landing_region]."
+
+/obj/effect/overmap/visitable/sector/exoplanet/moghes/generate_habitability()
+	return HABITABILITY_IDEAL
+
+/obj/effect/overmap/visitable/sector/exoplanet/moghes/generate_map()
+	if(prob(75))
+		lightlevel = rand(5,10)/10
+	..()
+
+/obj/effect/overmap/visitable/sector/exoplanet/moghes/generate_atmosphere()
+	..()
+	if(atmosphere)
+		atmosphere.remove_ratio(1)
+		atmosphere.adjust_gas(GAS_OXYGEN, MOLES_O2STANDARD, 1)
+		atmosphere.adjust_gas(GAS_NITROGEN, MOLES_N2STANDARD, 1)
+		if(landing_region == "Wasteland")
+			atmosphere.temperature = T0C + rand(40, 50)
+		else
+			atmosphere.temperature = T0C + rand(30, 40)
+		atmosphere.update_values()
+
+/obj/effect/overmap/visitable/sector/exoplanet/moghes/update_icon()
+	return
+
+/obj/effect/overmap/visitable/sector/exoplanet/moghes/generate_planet_image()
+	skybox_image = image('icons/skybox/lore_planets.dmi', "moghes")
+	skybox_image.pixel_x = rand(0,64)
+	skybox_image.pixel_y = rand(128,256)
+
+/obj/effect/overmap/visitable/sector/exoplanet/moghes/generate_ground_survey_result()
+	..()
+	switch(landing_region)
+		if("Untouched Lands")
+			ground_survey_result += "<br>Low levels of radioactivity detected in the soil"
+			ground_survey_result += "<br>Signs indicate soil is undergoing nutrient depletion"
+			if(prob(40))
+				ground_survey_result += "<br>High quality natural fertilizer found in subterranean pockets"
+			if(prob(40))
+				ground_survey_result += "<br>High nitrogen and phosphorus contents of the soil"
+			if(prob(40))
+				ground_survey_result += "<br>Analysis indicates low contaminants of the soil"
+			if(prob(40))
+				ground_survey_result += "<br>Soft clays detected, composed of quartz and calcites"
```

Comment:
ground survey for moghes looks wrong
should not be random still
and it includes stuff like low contaminants in soil - wasn't it like, nuked
so it looks like just copied from parent or something